test/scrabBag_test.js

```diff
@@ -0,0 +1,67 @@
+import { expect } from 'chai'
```

Comment:
in general keep an eye on using ```let``` vs ```const``` 

```let``` should only be used for temp variables that will be altered later. Everythign else is immutable so you need ```const```